.github/workflows/tools-image.yaml

```diff
@@ -0,0 +1,63 @@
+name: Build and Publish Tools Image
+
+on:
+  push:
+    branches:
+      - main
+      - "release-*"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+env:
+  IMG_REGISTRY_HOST: quay.io
+  IMG_REGISTRY_ORG: kuadrant
+  IMG_REGISTRY_REPO: mgc-tools
+  MAIN_BRANCH_NAME: main
```

Comment:
Is this repo in place?

Reply:
No, not yet. Does it need to be created manually before this action runs?